docs/sources/features/datasources/azuremonitor.md

```diff
@@ -274,6 +274,40 @@ There are also some Grafana variables that can be used in Azure Log Analytics qu
 
 - `$__interval` - Grafana calculates the minimum time grain that can be used to group by time in queries. More details on how it works [here]({{< relref "../../reference/templating.md#interval-variables" >}}). It returns a time grain like `5m` or `1h` that can be used in the bin function. E.g. `summarize count() by bin(TimeGenerated, $__interval)`
 
+### Templating with Variables for Azure Log Analytics
+
+Any Log Analytics query that returns a list of values can be used in the `Query` field in the Variable edit view. There is also one Grafana function for Log Analytics that returns a list of workspaces.
+
+Check out the [Templating]({{< relref "../../reference/templating.md" >}}) documentation for an introduction to the templating feature and the different
+types of template variables.
+
+| Name                                               | Description                                                                                            |
+| -------------------------------------------------- | ------------------------------------------------------------------------------------------------------ |
+| _workspaces()_                                     | Returns a list of workspaces for the default subscription.                                             |
+| _workspaces(12345678-aaaa-bbbb-cccc-123456789aaa)_ | Returns a list of workspaces for the specified subscription (the parameter can be quoted or unquoted). |
+
+Example variable queries:
```

Comment:
I recommend reformatting this bulleted list into a table.

Reply:
This is what it looks like as a table:

![image](https://user-images.githubusercontent.com/434655/77012436-3e413000-696e-11ea-9a59-3e408a649198.png)

Reply:
Ah, something weird with the quotes there. One sec.

Reply:
It was the prettier tool we use that automatically escapes dollar signs. https://github.com/prettier/prettier/issues/6213 

I disabled prettier just for this table so that no-one else on the team accidentally adds those extra slashes back.